### PR TITLE
Update web site documentation for latest kpod

### PR DESCRIPTION
**Pull request by rhatdan**
Add missing commands to transfer page and add
links for demos to README.md page.

Fixup bash completions for new commands.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,28 +37,29 @@ The plan is to use OCI projects and best of breed libraries for different aspect
...
+| [kpod-cp(1)](/docs/kpod-cp.1.md)                     | Copy files/folders between a container and the local filesystem           ||
+| [kpod-diff(1)](/docs/kpod-diff.1.md)                 | Inspect changes on a container or image's filesystem                      ||
+| [kpod-export(1)](/docs/kpod-export.1.md)             | Export container's filesystem contents as a tar archive                   ||
+| [kpod-history(1)](/docs/kpod-history.1.md)           | Shows the history of an image                                             ||
+| [kpod-images(1)](/docs/kpod-images.1.md)             | List images in local storage                                              |[...](https://asciinema.org/a/133649)|
+| [kpod-info(1)](/docs/kpod-info.1.md)                 | Display system information                                                ||
+| [kpod-inspect(1)](/docs/kpod-inspect.1.md)           | Display the configuration of a container or image                         ||
+| [kpod-load(1)](/docs/kpod-load.1.md)                 | Load an image from docker archive or oci                                  |[...](p8kOaexEhEa20P1KLZ3L5X4g)|
```
Comment (TomSweeneyRedHat) [inline README.md]: bad link for the demo for this one.

File: README.md [hunk 2]
```diff
@@ -37,28 +37,29 @@ The plan is to use OCI projects and best of breed libraries for different aspect
...
+| [kpod-mount(1)](/docs/kpod-mount.1.md)               | Mount a working container's root filesystem                               ||
+| [kpod-pull(1)](/docs/kpod-pull.1.md)                 | Pull an image from a registry                                             |[...](https://asciinema.org/a/lr4zfoynHJOUNu1KaXa1dwG2X)|
+| [kpod-push(1)](/docs/kpod-push.1.md)                 | Push an image to a specified destination                                  |[...](https://asciinema.org/a/133276)|
+| [kpod-rename(1)](/docs/kpod-rename.1.md)             | Rename a container                                                        ||
+| [kpod-rmi(1)](/docs/kpod-rmi.1.md)                   | Removes one or more images                                                |[...](https://asciinema.org/a/133799)|
+| [kpod-save(1)](/docs/kpod-save.1.md)                 | Saves an image to an archive                                              |[...](https://asciinema.org/a/kp8kOaexEhEa20P1KLZ3L5X4g)|
+| [kpod-stats(1)](/docs/kpod-stats.1.md)               | Display a live stream of one or more containers' resource usage statistics|[...](https://asciinema.org/a/133418)|
```
Comment (TomSweeneyRedHat) [inline README.md]: The demo here appears to be one for inspect, not stats.

File: README.md [hunk 3]
```diff
@@ -37,28 +37,29 @@ The plan is to use OCI projects and best of breed libraries for different aspect
...
+| [kpod-tag(1)](/docs/kpod-tag.1.md)                   | Add an additional name to a local image                                   |[...](https://asciinema.org/a/133803)|
+| [kpod-umount(1)](/docs/kpod-umount.1.md)             | Unmount a working container's root filesystem                             ||
+| [kpod-version(1)](/docs/kpod-version.1.md)           | Display the Kpod version information                                      |[...](https://asciinema.org/a/mfrn61pjZT9Fc8L4NbfdSqfgu)|
```
Comment (TomSweeneyRedHat) [inline README.md]: Should the 'K' in "Display the Kpod version" be uppercased?

File: README.md (hunk 1 above)
Comment (TomSweeneyRedHat) [inline README.md]: I find the '...' in the demo column to be light and hard to see.  I think the word demo for the link would be better.  Also many of the demo's on the asciinema.org site don't have a title.  Can we go back and touch that up?

Comment (rhatdan) [inline README.md]: No idea.  @umohnani8 Could you try to make sure your demos have a title.

Comment (rhatdan) [inline README.md]: Fixed

File: README.md (hunk 2 above)
Comment (rhatdan) [inline README.md]: Fixed

File: README.md (hunk 3 above)
Comment (rhatdan) [inline README.md]: Removed kpod

File: README.md (hunk 1 above)
Comment (rhatdan) [inline README.md]: Replaced ... with arrow,  Will attempt to get someone with better art skills to improve my arrow image.

**Comment by rhatdan** [comment]
Not seeing any Lint errors now.  @TomSweeneyRedHat @umohnani8 @mheon PTAL
No need for E2E tests since this is just docs.  Will update arrow png when I get a better one.

**Comment by rhatdan** [comment]
@umohnani8 @mheon @mrunalp @runcom PTAL

**Comment by mheon** [comment]
All the asciinema links work, and the docs changes look fine. LGTM.

**Comment by mrunalp** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (TomSweeneyRedHat) [inline README.md]: FWIW, the arrows look fine and dandy to me, thanks for changing/fixing.

**Pull request merged**